Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'jira-ruby', :group => :development
 
 group :test do
   gem 'rspec'
-  gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.9.0')
+  gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.12.0')
```

Comment:
Since we don't specify the z component, the y is allowed to float. So we're basically allowing versions >= 2.8 and less than 3